pravega/pravega
# Issue 2446: Update Documentation Links

**Change log description**

Updated links to documentation files/folders.

**Purpose of the change**
Fixes #2446

The documentation directory was changed from `docs/` to `documentation/src/docs`. For historical purposes, this occurred in https://github.com/pravega/pravega/pull/2169. Some links didn't get updated.

**How to verify it**

1. Check this out, run `./gradlew mkdocsServe` and explore the docs at http://127.0.0.1:8000. (A few things like 'click to edit' work on this branch.)
2. md docs visible at https://github.com/jdmaguire/pravega/tree/new-documentation-links repo. The links in `Quick Start` work in that but not currently on Pravega master.
